src/Validation/Validator.php

```diff
@@ -30,6 +30,12 @@
  */
 class Validator implements ArrayAccess, IteratorAggregate, Countable
 {
+    /**
+     * Used to flag nested rules created with addNested() and addNestedMany()
+     *
+     * @var string
+     */
+    const NESTED = '_nested';
```

Comment:
I can't grasp this PR but is it made sure that no valdidator called _nested creates an issue? Use __NESTED maybe?